Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/PropertyRowWidget.cpp

```diff
@@ -1110,6 +1136,18 @@ namespace AzToolsFramework
         m_leftHandSideLayout->activate();
     }
 
+    void PropertyRowWidget::OnClickedToggleButton(bool checked)
+    {
+        if ((m_expanded && !checked) || (!m_expanded && checked))
```

Comment:
nit: arguably easier to read as `m_expanded != checked`

Reply:
Thanks, good catch!